app/api/bookyt/entities/base.rb

```diff
@@ -0,0 +1,10 @@
+module Bookyt
+  module Entities
+    class Base < Grape::Entity
+      format_with(:iso_timestamp) do |dt|
+        dt ? dt.iso8601 : nil
+      end
+      format_with(:float, &:to_f)
```

Comment:
I think casting to a string would be the save option. But I also see that it might need some parsing steps on the consuming client. But probably that explicit parsing is exactly what we want, because this would be the step where the rounding issues with float would show up.